ui/src/components/Table/StripedTable.tsx

```diff
@@ -48,6 +43,13 @@ const StripedTable = (props: {
     }
   }, [activePage, rows, rowsPerPage]);
 
+  if (activePageRows.length === 0) {
+    return (
+      <div className="striped-table empty">
+        <p className="p2">This table contains on data...yet..</p>
```

Comment:
I think you mean "This table doesn't contain data yet..." or something like that ?

Reply:
pshh yes. Haha. Thanks!